docs/quick.md

```diff
@@ -164,10 +184,119 @@ meets the following prerequisites:
     NAME               HOSTNAME           ALLOCATED   FREE     CAPACITY   STATUS   AGE
     cspc-stripe-vn92   worker1            260k        19900M   19900M     ONLINE   2m17s
     ```
-11. Once your pool has came online. Follow steps from 
-    [here](https://github.com/openebs/cstor-csi#provision-a-cstor-volume-using-openebs-cstor-csi-driver) to provision a volume and deploy an app.
-    Please note, we already have created a pool using CSPC and we can skip the pool creation step explained there.
-     
+11. Once your pool has came online, we are ready with volume provisioning.
+    Create a Storage Class to dynamically provision volumes using OpenEBS CSI provisioner.
+    A sample storage class looks like:
+
+   ```yaml
+   kind: StorageClass
+   apiVersion: storage.k8s.io/v1
+   metadata:
+     name: openebs-csi-cstor-stripe
+   provisioner: cstor.csi.openebs.io
+   allowVolumeExpansion: true
+   parameters:
+     cas-type: cstor
+     cstorPoolCluster: cspc-stripe
+     replicaCount: "1"
+   ```
+
+   Create StorageClass using above example
+
+   ```bash
+    $ kubectl apply -f csi-cstor-sc.yaml
+   ```
+
+   You will need to specify the correct cStor CSPC from your cluster
+   and specify the desired `replicaCount` for the volume. The `replicaCount`
+   should be less than or equal to the max pools available.
+
+12. Create a PVC yaml with the StorageClass name
+
+    ```yaml
+    kind: PersistentVolumeClaim
+    apiVersion: v1
+    metadata:
+      name: demo-cstor-vol
+    spec:
+      storageClassName: cstor-csi-stripe
```

Comment:
Above storageclass named it as `openebs-csi-cstor-stripe` but here `cstor-csi-stripe`. We need to update it

Reply:
done